embark.el

```diff
@@ -751,23 +752,21 @@ UPDATE is the indicator update function."
                 (embark--read-key-sequence update)))
          (cmd (let ((overriding-terminal-local-map keymap))
                 (key-binding key))))
-    (setq cmd
```

Comment:
This was silly, wasn't it? I think there's probably a historical reason it evolved to this, but I don't remember.

Reply:
Embark evolved a lot